Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python
+
+WORKDIR /opt/service
+
+RUN pip install poetry
+RUN poetry install
+RUN uvicorn comprehensive_microservice.api_service:app

```

Comment:
you cannot RUN uvicorn inside the docker file
let's specify uvicorn run command in the docker-compose.yml file (google docker compose service command override)